litmus-portal/frontend/src/pages/ApplicationDashboard/index.tsx

```diff
@@ -854,7 +860,7 @@ const DashboardPage: React.FC = () => {
                         }}
                         panel_group_id={panelGroup.panel_group_id}
                         panel_group_name={panelGroup.panel_group_name}
-                        panels={panelGroup.panels}
+                        panels={panelGroup.panels ?? panelGroup.panels}
```

Comment:
May we revert this change. It does not add any advantage.

Reply:
Sometimes the UI crashes if panels are undefined due to some illegal character via translation this this check has been added, will add a ternary operator instead.

Reply:
Please suggest if you have any better workarounds ? @whitetiger1399

Reply:
May replace it with `{ panelGroup.panels ?? [] }`

Reply:
Done with previous commit.